### PR TITLE
Debris follow fix 2: Electric Boogaloo

### LINKED ISSUE #6058: The immovable DUCKs in Centcom Testing trigger ghost auto-follow debris
Issue reported from  Round ID: 23880 (Yogstation 13)

Reporting client version: 512.1471

There's no point in having the button if it leads to some uninteresting motionless DUCK every time. Same problem occurs with the unmoving immovable rod in the wizard ruin, except not every round.
scrnshot1
Speaking of uninteresting objects of interest, last I checked, supermatter monitoring apps now detect the actual station supermatter as crystal (#)2 because of that sword being (#)1.

### LINKED ISSUE #7318: "The immovable DUCKs in Centcom Testing trigger ghost auto-follow debris"
Auto-follow debris annoyance still present since an issue was made by Sniblet in June.
https://github.com/yogstation13/Yogstation/issues/6058#issue-459630896

### PR DESCRIPTION
**Pull request by Mr-Space**
### Intent of your Pull Request
Forgot to update my fork

Fixes #7318 
Fixes #6058
probably..
also deletes some immovable rods from wiznerds because they also cause to auto-follow to get the retard
#### Changelog

:cl:  

rscdel: Wiznerds no longer own immovable rods
bugfix: Ducks probably don't trigger the auto-follow
experimental: I don't know how to code
/:cl:

### CODE REVIEW

**Comment by Lostlarry** [comment]
what about replacing the rod with something that looks like a rod?

**Comment by Mr-Space** [comment]
This is only to fix the debris follow, ain't planning on that

**Comment by Mr-Space** [comment]
reminder for myself to dab on issue #6058 if this works

File: yogstation/code/modules/events/immovable_duck.dm [hunk 1]
```diff
@@ -32,6 +32,7 @@
 /obj/effect/immovablerod/duck/Initialize()
 	. = ..()
 	SpinAnimation(24,-1)
+	SSaugury.unregister_doom(src)
```
Comment (alexkar598) [inline yogstation/code/modules/events/immovable_duck.dm]: Instead, make a variable that can be set by mappers on immovable rods to make it not auto follow

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: yogstation/code/modules/events/immovable_duck.dm (hunk 1 above)
Comment (Mr-Space) [reply yogstation/code/modules/events/immovable_duck.dm]: Sure, if i knew how.

Comment (Lostlarry) [reply yogstation/code/modules/events/immovable_duck.dm]: just add var to immovable rod called something like notdebris and then on immovable rod initialise if(notdebris == true ){SSaugury.unregister_doom(src)}

**Comment by Mr-Space** [comment]
I'm workin on a var version to this so stay tuned.

**Comment by Mr-Space** [comment]
travis committed sudoku

**Review by alexkar598 — DISMISSED** [review]
Okay now readd the rods to the two maps and set the notdebris var to one in the map editor

**Comment by Mr-Space** [comment]
> 
> 
> Okay now readd the rods to the two maps and set the notdebris var to one in the map editor

Done, should be merge ready if i did the thing right

**Pull request merged**